app/domain/content/query.rb

```diff
@@ -160,7 +163,12 @@ def set(instance_variable, value)
     end
 
     def builder(verify_presence:)
-      yield if verify_presence.present?
+      if verify_presence.is_a?(Array)
+        yield unless verify_presence.empty?
+      else
+        yield if verify_presence.present?
```

Comment:
`verify_presence` is present if it’s not blank. To be blank is to be false, empty, or a whitespace string. Therefore we don't need the condition:

```rb
def builder(verify_presence:)
  yield if verify_presence.present?
end
```

Reply:
Addressed in ca4c313e67ee6c298405c6c65eeea9c899e33b99